fdmi/plugins/fdmi_plugin_st.sh

```diff
@@ -92,22 +125,36 @@ do_some_kv_operations()
                     rc=22 # EINVAL to indicate the test is failed
                     echo "m0kv index del should fail, but did not"
 		}
+		if $interactive ; then echo "Press Enter to go ..." && read; fi
 
 	done
 	return $rc
 }
 
 start_fdmi_plugin()
 {
+	local fdmi_filter_fid=$1
+	local fdmi_plugin_ep=$2
+	local fdmi_output_file="${fdmi_filter_fid}.txt"
 	local rc=0
 
 	# We are using FDMI_PLUGIN_EP
-	MOTR_PARAM="-l ${lnet_nid}:$FDMI_PLUGIN_EP        \
+	MOTR_PARAM="-l ${lnet_nid}:$fdmi_plugin_ep        \
 		    -h ${lnet_nid}:$HA_EP -p $PROF_OPT    \
 		    -f $M0T1FS_PROC_ID                    "
 
-	"$M0_SRC_DIR/fdmi/plugins/fdmi_sample_plugin" $MOTR_PARAM -g $FDMI_FILTER_FID &
-	sleep 5
+	PLUGIN_CMD="$M0_SRC_DIR/fdmi/plugins/fdmi_sample_plugin $MOTR_PARAM -g $fdmi_filter_fid -s"
+
+	if $interactive ; then
+		echo "Please use another terminal and run this command:"
+		echo sudo ${PLUGIN_CMD}
```

Comment:
This is to echo the string, so no harm without double quote.
Actually, I intentionally do this without double quote, so users can easily copy and paste it to another terminal.